commercetools-models/src/main/java/io/sphere/sdk/products/queries/ProductByIdHead.java

```diff
@@ -0,0 +1,35 @@
+package io.sphere.sdk.products.queries;
+
+import io.sphere.sdk.expansion.ExpansionPath;
+import io.sphere.sdk.expansion.MetaModelReferenceExpansionDsl;
+import io.sphere.sdk.models.Identifiable;
+import io.sphere.sdk.products.Product;
+import io.sphere.sdk.products.expansion.ProductExpansionModel;
+import io.sphere.sdk.products.search.PriceSelectionRequestDsl;
+import io.sphere.sdk.queries.HeadDsl;
+import io.sphere.sdk.queries.MetaModelHeadDsl;
+
+import java.util.List;
+
+public interface ProductByIdHead extends MetaModelHeadDsl<Product, Product, ProductByIdHead, ProductExpansionModel<Product>>, PriceSelectionRequestDsl<ProductByIdHead> {
```

Comment:
The Head methods don't have expansion, price selection etc

Reply:
thanks, I followed the wrong PR about the price selection